cmd/kots/cli/get-config.go

```diff
@@ -234,6 +222,21 @@ func decryptGroups(clientset kubernetes.Interface, namespace string, groups []v1
 	return outGroups, nil
 }
 
+func extractString(item string) multitype.BoolOrString {
+	if item == "" {
+		return multitype.BoolOrString{}
+	}
+	decrypted, err := decryptString(item)
+	if err != nil {
+		fmt.Println(err)
+		return multitype.BoolOrString{} // don't fail if we can't decrypt
```

Comment:
why not fail?

Reply:
Could do, I just wasn't 100% sure what may break, currently we don't do anything with the error

Reply:
Let's fail here please so that customers aren't confused if their passwords were empty

Reply:
Ah I see that we weren't checking the error before, so this is fine